python/ray/data/impl/arrow_block.py

```diff
@@ -224,20 +224,18 @@ def to_pandas(self) -> "pandas.DataFrame":
         return self._table.to_pandas()
 
     def to_numpy(self, column: str = None) -> np.ndarray:
-        if not column:
-            raise ValueError(
-                "`column` must be specified when calling .to_numpy() "
-                "on Arrow blocks.")
-        if column not in self._table.column_names:
-            raise ValueError(
-                "Cannot find column {}, available columns: {}".format(
-                    column, self._table.column_names))
-        array = self._table[column]
-        if array.num_chunks > 1:
-            # TODO(ekl) combine fails since we can't concat ArrowTensorType?
-            array = array.combine_chunks()
-        assert array.num_chunks == 1, array
-        return self._table[column].chunk(0).to_numpy()
+        if column is None:
+            columns = self._table.column_names
+        else:
+            if column not in self._table.column_names:
+                raise ValueError(
+                    "Cannot find column {}, available columns: {}".format(
+                        column, self._table.column_names))
+            columns = [column]
+        arrays = [self._table[col].to_numpy() for col in columns]
```

Comment:
@ericl I didn't have any issues with tensor columns when using this direct `ChunkedArray.to_numpy()` call, so I'm opting for it for now since it's simpler. Do you remember why it didn't work before?

Reply:
I think it will still fail if there are multiple tensor chunks, but I don't remember when this happens.

Reply:
I haven't been able to trigger it so far.